LoggingKata.Test/TacoParserTests.cs

```diff
@@ -8,22 +8,47 @@ public class TacoParserTests
         [Fact]
         public void ShouldDoSomething()
         {
-            // TODO: Complete Something, if anything
+            // Currently not used

```

Comment:
Unused code can be removed (unless it has a future purpose, in which case `// TODO: ` can be used to indicate)